tests/e2e/tidbcluster/tidbcluster.go

```diff
@@ -705,6 +705,23 @@ var _ = ginkgo.Describe("[tidb-operator] TiDBCluster", func() {
 		err = oa.WaitForTidbClusterReady(tc, 30*time.Minute, 15*time.Second)
 		framework.ExpectNoError(err, "Expected TiDB cluster scaled in and ready")
 	})
+
+	ginkgo.It("TidbMonitor: Deploying and checking monitor", func() {
+		cluster := newTidbClusterConfig(e2econfig.TestConfig, ns, "monitor-test", "admin", "")
+		cluster.Resources["pd.replicas"] = "1"
+		cluster.Resources["tikv.replicas"] = "1"
+		cluster.Resources["tidb.replicas"] = "1"
+		oa.DeployTidbClusterOrDie(&cluster)
+		oa.CheckTidbClusterStatusOrDie(&cluster)
+
+		tc, err := cli.PingcapV1alpha1().TidbClusters(cluster.Namespace).Get(cluster.ClusterName, metav1.GetOptions{})
+		framework.ExpectNoError(err, "Expected get tidbcluster")
+
+		tm := newTidbMonitor("monitor", tc.Namespace, tc, true, false)
+		oa.DeployTidbMonitorOrDie(tm)
+		oa.CheckTidbMonitorOrDie(tm)
```

Comment:
inline these two functions

Current `operatorActions` is too complicated. We should not add more methods to it.

Reply:
if you think the logic of `CheckTidbMonitor` will be used by other e2e tests, you can add a simple utility function for it.

Reply:
updated